rapid7_insightidr/help.md

```diff
@@ -36,6 +36,102 @@ Example input:
 
 ### Actions
 
+#### Advanced Query
+
+This action is used to realtime query into Insight IDR logs.
```

Comment:
This sentence is awkward. Maybe:

This action is used to run realtime queries into Insight IDR logs.

Reply:
Fixed